### PR TITLE
Feature/two-files-input

### PR DESCRIPTION
**Pull request by FilipKolodziejczyk**

### CODE REVIEW

File: examples/1.txt [hunk 1]
```diff
@@ -1,22 +1,13 @@
...
 1 0 1 0 0
 0 1 0 1 0
 1 0 1 0 1
 0 0 0 1 0
-5
-0 1 0 0 0
-1 0 1 0 0
-0 1 0 1 0
-0 0 1 0 1
-0 0 0 1 0
```
Comment (jswistak) [inline examples/1.txt:22]: shouldn't we have space at the end?

File: examples/3.txt [hunk 2]
```diff
@@ -1,28 +1,18 @@
...
-7 1 
-4
-1 7 7 27 
-7 1 25 14 
-7 25 1 6 
-27 14 6 1 
+7 1
+
+
+
```
Comment (jswistak) [inline examples/3.txt:7]: why so many spaces?

File: examples/1.txt (hunk 1 above)
Comment (FilipKolodziejczyk) [reply examples/1.txt:22]: Not really

File: examples/3.txt (hunk 2 above)
Comment (FilipKolodziejczyk) [reply examples/3.txt:7]: I will fix it

**Pull request merged**